0009-salt-working-groups.md

```diff
@@ -0,0 +1,121 @@
+
+# Working Groups 
+
+A Working Group is a small group of individuals who come together with a common goal and work towards achieving that goal within a predetermined amount of time. It’s an opportunity for the Salt community to lead Salt projects and be part of the process in its entirety. 
+
+## Eligibility to participate 
+
+Salt community members are eligible to be part of a working group. A community member is one who is interested in working on the Salt project exhibited by raising either 1 or more PR(s), issue(s) or SEP(s) (Salt Enhancement Proposal).
+
+In addition, a working group applicant is a Subject Matter Expert (SME) in areas such as Windows, Cloud, Networking or SSH 
+
+If you want to lead or be part of a different working group then the ones mentioned above, Salt would be happy to include that as part of initial working groups, provided Salt receives enough interest from others to participate. 
+
+## Application Process
+
+Salt community members indicate their interest to be part of one or more working group(s) by filling out an application consisting of simple 4-5 question survey; Applicants indicate their preference for the type of working group(s) they want to join and the role they want to play: Captain or facilitator. If an application is received past the due date then the applicant will automatically be considered for the next term of the working groups. 
+
+## Election 
+
+Salt will conduct an election to choose the working group members in the following scenarios: 
+- When the number of applicants interested in joining the working groups exceeds the number that is ideal for a working group (3 to 9 members) 
+- 2 or more community members indicate their interest in being captain
+- If Salt doesn’t receive any eligible applicants for captain role, a member of the Salt open/core will be elected to be captain of that working group. 
+
+The rules for the election process are:  
+- Election for the first working group captain will be internal and voting will be done by the Salt core team. 
+- Elections for all subsequent working group captains and members will be voted on by the outgoing members of the current working group
+
+## Timelines 
+
+Based on the applications submitted, Salt chooses the final working group members 
+
+Below is the approximate timeline for how long each working group term will last. The actual timeline may differ. For example, the Testing and Release Working Group might decide to convene immediately after a major release and dissolve after the next major release. 
+
+![Working Group Timelines](./diagrams/Working-group-timelines.svg)
+
+
+
+*T0 is feature freeze date of current major release
+ T1 is feature freeze date of next major release
+
+
+Working Groups will meet once a month for 60 mins. Below are some guidelines for working group participants, although each of these groups may operate a little differently depending on their needs and workflow.
```

Comment:
Most commonly, I have seen these types of meeting scheduled no less than once a week. 

Will these be recorded? Will recordings be public? Does one need to be a member of the group to attend?

Reply:
To clarify, I changed it as: Working Groups will meet not less than once a month for 60 mins

Also we make it clear that Captains can hold as many discussions/meetings as needed. Below is an excerpt
- The captain will attend monthly working group meetings for their respective working group. SaltStack will set these up, but SaltStack will hold no restriction on any additional meetings the captain may deem necessary

The meetings are being recorded and made available on Youtube. Meetings are open for anyone to attend and meeting links are shared publicly. I have added the info now to this SEP